content-src/components/DiscoveryStreamComponents/DSTextPromo/_DSTextPromo.scss

```diff
@@ -0,0 +1,86 @@
+.ds-text-promo {
+  display: flex;
+  max-width: 744px;
+  margin: 16px auto;
+
+  img {
+    width: 40px;
+    height: 40px;
+    margin: 0 12px 0 0;
+    border-radius: 4px;
+  }
+
+  .text {
+    line-height: 24px;
+    margin: -4.5px 0 0;
+  }
+
+  header {
+    @include dark-theme-only {
+      color: $grey-10;
+    }
+
+    font-weight: 600;
+    font-size: 15px;
+  }
+
+  .subtitle {
+    @include dark-theme-only {
+      color: $grey-40;
+    }
+
+    font-size: 13px;
+    margin: 0;
+    color: $grey-50;
+  }
+}
+
+.ds-chevron-link {
+  color: $blue-60;
+  display: inline-block;
+  outline: 0;
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &:active {
+    @include dark-theme-only {
+      color: $blue-50;
+    }
+
+    color: $blue-70;
+
+    &::after {
+      @include dark-theme-only {
+        background-color: $blue-50;
+      }
+
+      background-color: $blue-70;
+    }
+  }
+
+  &:focus {
+    @include dark-theme-only {
+      box-shadow: 0 0 0 2px $grey-80, 0 0 0 5px $blue-50-50;
+    }
+
+    box-shadow: 0 0 0 2px $white, 0 0 0 5px $blue-50-50;
+    border-radius: 2px;
+  }
+
+  &::after {
+    @include dark-theme-only {
+      background-color: $blue-40;
+    }
+
+    content: ' ';
+    mask: url('resource://activity-stream/data/content/assets/glyph-caret-right.svg') 0 -8px no-repeat;
```

Comment:
nit: should consider global variable [$image-path] here, help keep assets reference at one place (https://github.com/mozilla/activity-stream/blob/master/content-src/styles/_variables.scss#L153)